docs/scripts_md/tarchive_validation.md

```diff
@@ -0,0 +1,74 @@
+# NAME
+
+tarchive\_validation.pl -- Validates the tarchive against the one inserted in
+the LORIS database.
+
+# SYNOPSIS
+
+perl tarchive\_validation.pl `[options]`
+
+Available options are:
+
+\-profile     : name of the config file in ../dicom-archive/.loris-mri
+
+\-reckless    : upload data to the database even if the study protocol
+                is not defined or if it is violated
+
+\-globLocation: loosen the validity check of the tarchive allowing for
+                the possibility that the tarchive was moved to a
+                different directory
+
+\-newScanner  : boolean, if set, register new scanners into the database
+
+\-verbose     : boolean, if set, run the script in verbose mode
+
+# DESCRIPTION
+
+The program does the following validations:
+
+\- Verification of the DICOM study archive given as an argument to the script
+using the checksum from the one inserted in the database
+
+\- Verification of the PSC information using whatever field containing the site
+string (typically, the patient name or patient ID)
+
+\- Verification/determination of the ScannerID of the DICOM study archive
+(optionally creates a new scanner entry in the database if necessary)
+
+\- Optionally, creation of candidates as needed and standardization of gender
+information when creating the candidates (DICOM uses M/F, LORIS database uses
+Male/Female)
+
+\- Check of the CandID/PSCID match. It's possible that the CandID exists, but
+that CandID and PSCID does not correspond to the same candidate. This would
+fail further down silently, so we explicitly check that this information is
+correct here.
+
+\- Validation/Obtention of the SessionID
```

Comment:
lol "obtention"; I googled it to make sure it is english (I thought it was a french translation)...
but it looks like it is used oin english, @driusan ?